setup.py

```diff
@@ -40,4 +40,5 @@
                         'webapp2>=2.5.2',
                         'WebOb>=1.4',
                         'PyYAML>=3.11',
-                        'Werkzeug>=0.10',],)
+                        'Werkzeug>=0.10',
+                        'google-python-cloud-debugger>=1.2'],)
```

Comment:
Let's not add this to setup.py requirements and instead add it to the Dockerfile. The debugger isn't supposed to be installed in local dev mode, right? If we add it here instead of in the Dockerfile, it will be installed and activated in dev mode.